corehq/apps/saved_reports/models.py

```diff
@@ -241,6 +241,22 @@ def query_string(self):
 
         return urlencode(params, True)
 
+    @property
+    @memoized
+    def serialized_filters(self):
+        """
+        converts any filter values that are objects to strings
```

Comment:
nit: this doc string is slightly misleading. i think it only converts dates, right?